Makefile

```diff
@@ -29,15 +26,6 @@ clean:
 	@rm -rf $(BUILDLIST)
 	@rm -rf $(BIN)
 
-# mock autogeneration for interface testing
-mock: mock-internal mock-pkg
-
-mock-internal:
-	@$(MOCK) --dir=./internal --all --output=./$(MOCKS)
-
-mock-pkg:
-	@$(MOCK) --dir=./pkg --all --output=./$(MOCKS)
-
```

Comment:
Maybe I grossly misunderstood what was going on but... from what I could tell we did not use any of the generated mocks internally. Additionally, since they were in `internal` they couldn't be used externally. So, I removed the unused mocks and got rid of mockery from this project.

For context, I did try first updating our use of mockery here to work with v3 (i.e., the new `mockery.yaml` format) but I realized that you can't mock code in `pkg`. At least, you can't unless that code is published and available via go get. Since there is no code to be mocked in internal I figured it was best to remove mockery for now.